openstack/compute/v2/extensions/quotasets/urls.go

```diff
@@ -8,14 +8,14 @@ func resourceURL(c *gophercloud.ServiceClient) string {
 	return c.ServiceURL(resourcePath)
 }
 
-func getURL(c *gophercloud.ServiceClient, tenantID string) string {
+func getDetailURL(c *gophercloud.ServiceClient, tenantID string) string {
```

Comment:
I apologize - I didn't explain this well enough. What I meant was to create a new URL in `urls.go` and keep the existing `getURL`.

So `getURL` would remain unchanged and `getDetailURL` would look like:

```go
func getDetailURL(c *gophercloud.ServiceClient, tenantID string) string {
        return c.ServiceURL(resourcePath, tenantID, "detail")
}
```

By doing this, the `Get` function in `requests.go` remains unchanged and `GetDetail` will now look like:

```go
// GetDetail returns detailed public data about a previously created QuotaSet.
func GetDetail(client *gophercloud.ServiceClient, tenantID string) GetDetailResult {
	_, res.Err = client.Get(getDetailURL(client, tenantID), &res.Body, nil)
 	return res
 }
```